src/backoffice/views.py

```diff
@@ -123,3 +123,48 @@ class EventProposalManageView(ProposalManageView):
     model = EventProposal
     template_name = "manage_eventproposal.html"
 
+
+class MerchandiseOrdersView(BackofficeViewMixin, ListView):
+    template_name = "orders_merchandise.html"
+
+    def get_queryset(self, **kwargs):
+        camp_prefix = 'BornHack {}'.format(datetime.now().strftime('%Y'))
```

Comment:
And here I would then do:
`camp_prefix = 'BornHack {}'.format(timezone.now().year)`